generators/example/src/main/scala/Configs.scala

```diff
@@ -247,3 +253,6 @@ class RV32BoomAndRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithRV32 ++
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   new freechips.rocketchip.system.BaseConfig)
+
+class DualCoreRocketL2Config extends Config(
```

Comment:
Can we also add a config that sets parameters in the L2 config?
`Rocket4BankL2` or `Rocket1MBL2` and just have them build in CI?
I think its important to test the number of banks and the number of memory channels. Having both of those parameters set has caused problems in the past and I think was the reason for Howie's rocket-chip PR.

Reply:
Sure. Thats fine with me.